dev/flux/src/index.ts

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```

Comment:
I _think_ you can specify a binary file to run in the pkg json rather than adding this. Then the cmd would become `npx @prosopo/flux ...` https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bin

no problem with doing both too tho!

Reply:
You need both it seems :)